tests/data/tools/yaml2nxdl_test_data/NXfilelineError1.yml

```diff
@@ -1,14 +1,14 @@
-#test case for attributes 
+#test case for attributes
 doc: documentation no. 0
 symbols:
   doc: documentation no. 1
   testnamesymbol: test description of symbol
 category: application
 extends: my_test_extend
```

Comment:
shouldn't be needed anymore

Reply:
I checked why the test_yaml2nxdl didn't fail after the changes you made to the "extends" keyword. I found it to be okay and you can safely remove line 7